### DISCUSSION TITLE
Oh My Posh''s default theme

### DISCUSSION THREAD

**Discussion by RadioactivePlays** [JanDeDobbeleer/oh-my-posh#4211] [Troubleshoot]
I really liked the default theme so where can I find the source code of this theme so i can tweak it a litle bit to my like?

**Comment by JanDeDobbeleer** [comment]
The default is embedded in the executable. You can export it like any other theme file. https://ohmyposh.dev/docs/installation/customize#adjust-a-theme

**Comment by RadioactivePlays** [comment]
Thank you.
